src/main/java/org/apache/commons/io/FileUtils.java

```diff
@@ -2618,15 +2615,14 @@ public static String readFileToString(final File file, final Charset charsetName
     /**
      * Reads the contents of a file into a String. The file is always closed.
      *
-     * @param file     the file to read, must not be {@code null}
+     * @param file the file to read, must not be {@code null}
      * @param charsetName the name of the requested charset, {@code null} means platform default
      * @return the file contents, never {@code null}
      * @throws NullPointerException if file is {@code null}.
-     * @throws FileNotFoundException if the file does not exist, is a directory rather than a regular file, or for some
-     *         other reason cannot be opened for reading.
-     * @throws IOException if an I/O error occurs.
+     * @throws IOException if an I/O error occurs, including when the file does not exist, is a directory rather than a
+     *         regular file, or for some other reason why the file cannot be opened for reading.
      * @throws java.nio.charset.UnsupportedCharsetException thrown instead of {@link java.io
-     * .UnsupportedEncodingException} in version 2.2 if the named charset is unavailable.
+     *                                                      .UnsupportedEncodingException} in version 2.2 if the named charset is unavailable.
```

Comment:
Hi @chkpnt 
Thank you for your PR.
Please don't change the formatting, it makes it harder to review the actual changes.

Reply:
Tried to apply the boys scouting rule for the sake of consistency... (sometimes the javadocs are aligned, sometime not). But yes, it was an unsolicited change, sorry for that!

(IHMO formatting shouldn't be something handled manually, I can recommend the integration of something like spotless in the build chain. An other issue I had was that my IDE added the new imports with an wildcard. As I didn't saw any other wildcard imports, I thought that might be undesired, so I had to change the import to single imports manually.)